src/connections.jl

```diff
@@ -71,7 +70,7 @@ mutable struct Connection
     semaphore::Semaphore
 
     "Current AsyncResult, if active"
-    async_result  # ::Union{AsyncResult, Nothing}, would be a circular reference
+    async_result::Any  # ::Union{AsyncResult, Nothing}, would be a circular reference
```

Comment:
Do we want this change or not (in general, for BlueStyle)? it can be turned off by passing `annotate_untyped_fields_with_any-false`

Reply:
i think we don't want this.
`BlueStyle()` should set `annotate_untyped_fields_with_any=false`

Reply:
Hmm yeah I think there's meaning in `::Any` that isn't always intended.